example/blzpack.c

```diff
@@ -323,7 +345,7 @@ decompress_file(const char *packedname, const char *newname, int use_checksum)
 		if (read_be32(header + 0 * 4) != 0x626C7A1AUL /* "blz\x1A" */
 		 || read_be32(header + 1 * 4) != 1
 		 || hdr_packedsize > max_packed_size
-		 || hdr_depackedsize > BLOCK_SIZE) {
+		 || hdr_depackedsize > effectiveblocksize) {
```

Comment:
I think this should be a separate error which informs the user that the blocksize was too small for the file.

Reply:
Done in the latest commit.

Reply:
Oh, wait, did you mean that the error should be different but the effectiveblocksize handling should be kept? I dropped that in this commit. Please tell me which way to do it. (Also tell me if you want me to use git rebase -i to squash the relevant commits together into one.)

Reply:
As I understood it, it now defaults to the size in DEFAULT_BLOCK_SIZE if no option. If the option is used, it uses that size. That seems okay.

Please feel free to rebase and clean up the commits, that should also get the CI working again as I (hopefully) fixed that now.

Reply:
Done!